### DISCUSSION TITLE
I need Fedora's rpm direct download site

### DISCUSSION THREAD

**Discussion by ToddAndMargo** [nxadm/rakudo-pkg#93] [Q&A] [answered]
Anyone know where rakudo's direct download site is for Fedora RPMs?

**Comment by nxadm** [comment, marked as answer]
You'll find direct links here (example for Fedora 36):
https://cloudsmith.io/~nxadm-pkgs/repos/rakudo-pkg/packages/?q=distribution%3Afedora+AND+distribution%3A36

**Reply by ToddAndMargo** [reply]
Thank you!
